docs/section12/Rule12-3.md

```diff
@@ -1,38 +1,58 @@
-
 # Receptacle - Rule 12-3
-
+**Schema Version** 0.0.37  
+**Primary Rule:** True  
 **Rule ID:** 12-3  
-**Rule Description:** User RMR Space ID in Proposed RMR?  
-**Rule Assertion:** Proposed RMR = User RMR  
-**Appendix G Section:** Section Table G3.1-12 Receptacles: Modeling Requirements for the Proposed design  
+**Rule Description:** When receptacle controls installed in spaces where not required by Standard 90.1 2019, Section 8.4.2 are included in the proposed building design, the hourly receptacle shall be reduced as specified in Standard 90.1-2019, Table G3.1 Section 12, Proposed Building Performance.   
```

Comment:
Recommended edit
"When receptacle controls are specified in the proposed building design for spaces where not required by Standard 90.1 2019 Section 8.4.2, the hourly receptacle schedule shall be reduced as specified in Standard 90.1-2019 Table G3.1 Section 12 Proposed Building Performance column.  "